src/mbgl/map/map_context.cpp

```diff
@@ -164,6 +164,11 @@ void MapContext::update() {
         return;
     }
 
+    // This time point is used to:
+    // - Calculate style property transitions;
+    // - Hint style sources to notify when all its tiles are loaded;
```

Comment:
This would be easier to look when we add some utests to check the different notifications from `View` at some point.